Tests/UnitTests/Core/Vertexing/IterativeVertexFinderTests.cpp

```diff
@@ -577,10 +574,7 @@ BOOST_AUTO_TEST_CASE(iterative_finder_test_athena_reference) {
   IPEstimator::Config ipEstimatorCfg(bField, propagator);
   IPEstimator ipEstimator(ipEstimatorCfg);
 
-  using ZScanSeedFinder = ZScanVertexFinder<BilloirFitter>;
-
-  static_assert(VertexFinderConcept<ZScanSeedFinder>,
```

Comment:
Why are we not asserting this anymore?

Reply:
`ZScanVertexFinder` is actually a seed finder and does not comply with the vertex finder interface

Reply:
lol yes that's true